src/components/table/OcTableFiles.vue

```diff
@@ -384,11 +383,12 @@ export default {
       this.$emit(EVENT_FILE_DROPPED, fileId)
     },
     addSelectedResource(file) {
-      const selectedResourceInResources = this.selectedResources.some(e => e.id === file.id)
+      const selectedResourceInResources = this.selection.some(e => e.id === file.id)
```

Comment:
Seeing this again now, what do you think about `isSelected` als name for the const?